### PR TITLE
fix: handling invalid client id error for bingads audience

### PR DESCRIPTION
**Pull request by shrouti1507**
# Description

we are encountering issue with bingAds bulk API returning client ID as empty string. 
We are handling this by checking the status file row name and eventually aborting all the events in importing state

## Linear Ticket

Resolves INT-1682

## Security

- [x] The code changed/added as part of this pull request won't create any security issues with how the software is being used.

### CODE REVIEW

File: router/batchrouter/asyncdestinationmanager/bing-ads/util.go [hunk 1]
```diff
@@ -111,6 +111,7 @@ func (b *BingAdsBulkUploader) populateZipFile(actionFile *ActionFileInfo, audien
 		actionFile.EventCount += 1
 		for _, uploadData := range data.Message.List {
 			clientIdI := newClientID(data.Metadata.JobID, uploadData.HashedEmail)
+			fmt.Println("######## clientIdI ###########", clientIdI)
```
Comment (cisse21) [inline router/batchrouter/asyncdestinationmanager/bing-ads/util.go:114]: use logger instead ?

Comment (shrouti1507) [reply router/batchrouter/asyncdestinationmanager/bing-ads/util.go]: removed

File: router/batchrouter/asyncdestinationmanager/bing-ads/bulk_uploader.go [hunk 2]
```diff
@@ -262,10 +263,26 @@ func (b *BingAdsBulkUploader) GetUploadStats(uploadStatsInput common.GetUploadSt
 		eventStatsResponse.StatusCode = response.StatusCode
 	}
 
+	// If the failedJobIds array has only one element and that is 0, that means that all the jobs have failed
```
Comment (shrouti1507) [inline router/batchrouter/asyncdestinationmanager/bing-ads/bulk_uploader.go:266]: this will be refactored

**Comment by shrouti1507** [comment]
We are closing this PR because we are handling this issue in a simpler way by just looking for list item error named rows. This will allow us to get rid of the problem with minimalistic change. cc @koladilip 
Raised PR : https://github.com/rudderlabs/rudder-server/pull/4429

**Pull request closed without merging**